apps/prairielearn/src/pages/instructorAssessmentUploads/instructorAssessmentUploads.html.ts

```diff
@@ -0,0 +1,295 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { z } from 'zod';
+
+import { Modal } from '../../components/Modal.html';
+import { JobSequenceSchema, UserSchema } from '../../lib/db-types';
+
+export const UploadJobSequenceSchema = z.object({
+  job_sequence: JobSequenceSchema,
+  start_date_formatted: z.string(),
+  user_uid: UserSchema.shape.uid,
+});
+type UploadJobSequence = z.infer<typeof UploadJobSequenceSchema>;
+
+export function InstructorAssessmentUploads({
+  resLocals,
+  uploadJobSequences,
+}: {
+  resLocals: Record<string, any>;
+  uploadJobSequences: UploadJobSequence[];
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head'); %>", resLocals)}
+      </head>
+      <body>
+        <script>
+          $(function () {
+            $('[data-toggle="popover"]').popover({ sanitize: false });
+          });
+
+          // make the file inputs display the file name
+          $(document).on('change', '.custom-file-input', function () {
+            this.fileName = $(this).val().replace(/\\\\/g, '/').replace(/.*\\//, '');
+            $(this).parent('.custom-file').find('.custom-file-label').text(this.fileName);
+          });
+        </script>
+        ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", resLocals)}
+        <main id="content" class="container-fluid">
+          ${renderEjs(
+            __filename,
+            "<%- include('../partials/assessmentSyncErrorsAndWarnings'); %>",
+            resLocals,
+          )}
+          ${resLocals.authz_data.has_course_instance_permission_edit
+            ? html`
+                ${UploadInstanceQuestionScoresModal({ csrfToken: resLocals.__csrf_token })}
+                ${UploadAssessmentInstanceScoresModal({ csrfToken: resLocals.__csrf_token })}
+              `
+            : ''}
+
+          <div class="card mb-4">
```

Comment:
In general I like to put each card in its own helper function, to keep the overall page structure clear. Here we could have a function like `AssessmentUploadCard()` (or whatever you want to call it).

Reply:
Moved this card to `AssessmentUploadCard()` in [1bbe13f](https://github.com/PrairieLearn/PrairieLearn/pull/9777/commits/1bbe13f56e186803b4fadcc23bd97315a6a6c7f5).